SalesforceCommerceCloud/pwa-kit
# Extract einstein from commerce-api folder

To integrate `commerce-sdk-react` in the retail react app, we'd like to remove the `commerce-api` folder and use the hooks provided from the library. However, the Einstein module lives in the `comerce-api` folder and we'd love to move it to somewhere else so we could remove the `commerce-api` folder.

## Goal

Move `einstein.js` out from `commerce-api` folder.

### Test drive

- Go to our reference app pwa-kit.mobify-storefront.com
- open chrome network tab, filter "cquotient"
- go through the user journey
  - home page -> PLP -> PDP -> add to cart -> cart -> checkout
 You should see the following einstein requests sent
<img width="293" alt="Screenshot 2023-03-12 at 9 29 59 PM" src="https://user-images.githubusercontent.com/10948652/224608738-c4a5e55a-df1b-43ef-9beb-2fa855afd974.png">
<img width="288" alt="Screenshot 2023-03-12 at 9 30 17 PM" src="https://user-images.githubusercontent.com/10948652/224608743-eb5d6515-d17f-461f-bce1-9fa894a4729b.png">

**You should see the same event being sent on this branch.**
